app/controllers/api_resource_comments_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ApiResourceCommentsController < ApiApplicationController
+  def show
+    resource_id = params['resource_id']
+    resource = Resource.find(resource_id)
+    comments = ResourceComment.where(resource:)
```

Comment:
can this be resource.comments? something more like the Rails way

Reply:
Agree, `resource.resource_comments`